packages/adapter-pg/src/pg.ts

```diff
@@ -87,7 +89,31 @@ class PgQueryable<ClientT extends StdClient | TransactionClient> implements Quer
     const { sql, args: values } = query
 
     try {
-      const result = await this.client.query({ text: sql, values: fixArrayBufferValues(values), rowMode: 'array' })
+      const result = await this.client.query(
+        {
+          text: sql,
+          values: fixArrayBufferValues(values),
+          rowMode: 'array',
+          types: {
+            // Error: packages/adapter-pg build: src/pg.ts(98,13): error TS2769: No overload matches this call.
```

Comment:
Would adding explicit type annotation to `customParsers` fix it?
`Record<number, types.TypeParser<string, unknown>>` should be the correct type.
Alternatively, maybe `as const` after `customParsers` declaration?

Reply:
Ah, I see the issue. Apparantly, `pg-types` types expect you to handle both binary and text protocol where versions, where as far as i can see, `pg` will ever pass only text version. We can put that into the comment instead, and then add assertion that `format === 'text'` before reaching out for our parsers.

Reply:
I did the update